src/cgame/cg_buildable.cpp

```diff
@@ -708,16 +708,9 @@ void CG_InitBuildables()
 
 			if ( cg_buildables[ buildable ].sounds[ j ].enabled )
 			{
-				if ( trap_FS_FOpenFile( filename, &f, fsMode_t::FS_READ ) > 0 )
-				{
-					//file exists so close it
-					trap_FS_FCloseFile( f );
-
-					cg_buildables[ buildable ].sounds[ j ].sound = trap_S_RegisterSound( filename, false );
-				}
-				else
+				if ( ( cg_buildables[ buildable ].sounds[ j ].sound = trap_S_RegisterSound( filename, false ) ) < 0 )
```

Comment:
Maybe assignment and comparison in one go is a bit hard to read

Reply:
yolo

Reply:
Done.